fleet_vehicle_service_activity/models/fleet_vehicle_log_services.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class FleetVehicleLogServices(models.Model):
+    _inherit = "fleet.vehicle.log.services"
+
+    @api.model
+    def _cron_manage_service_date(self):
+        params = self.env["ir.config_parameter"].sudo()
+        delay_alert_service = int(
+            params.get_param("hr_fleet.delay_alert_service", default=30)
+        )
+        date_today = fields.Date.today()
```

Comment:
I prefer to use `context_today` to prevent issues in countries where `UTC` is negative.
```suggestion
        date_today = fields.Date.context_today(self)
```